tests/RobotFramework/README.md

```diff
@@ -110,6 +110,22 @@ Checkout the [dev container instructions](../../docs/src/developer/DEV_CONTAINER
 
 3. Follow the console instructions and edit the `.env` file which was created by the `./bin/setup.sh` script
 
+    It is recommended to provide a local certificate authority (CA) signing certificate so that the system tests can utilize it to generate the device certificates. To use your own signing certificate you need to add the following keys which contain the base64 encoded contents of the CA key and public certificate.
+
+    For example, assuming you have the `~/tedge-ca.key` (private key) and the `~/tedge-ca.crt` public certificate, you can run the following on the console:
+
+    ```sh
+    cat ~/tedge-ca.key | base64
+    cat ~/tedge-ca.crt | base64
+    ```
+
+    Copy the output form above for the respective values in the following environment variables in the `.env` file (which will be used by the system tests).
+
+    ```sh
+    CA_KEY="{base64_encoded_tedge-ca.key}"
+    CA_PUB="{base64_encoded_tedge-ca.crt}"
```

Comment:
```suggestion
    CA_KEY="$(cat ~/tedge-ca.key | base64)"
    CA_PUB="$(cat ~/tedge-ca.crt | base64)"
```

Reply:
I've extended the instructions to a different section to also improve the easy of setting the required env variables.

Resolved by bb43d2f8d13cf481d2a89c029869a35a089deb25

Reply:
> That gives the read the wrong impression that the .env file is a script and can execute command (which it can't).

My bad, I though it was a shell script :-(